src/cli.ts

```diff
@@ -119,6 +122,117 @@ const latestPolkadotReleaseURL = async (
   }
 };
 
+// Convert functions
+// Read the input file
+async function readInputFile(ext: string, fPath: string): Promise<ConfigType> {
+  let json: object;
+  if (ext === "json" || ext === "js") {
+    json =
+      ext === "json"
+        ? JSON.parse(fs.readFileSync(`${fPath}`, "utf8"))
+        : await import(path.resolve(fPath));
+  } else {
+    throw Error("No valid extension was found.");
+  }
+  return json;
+}
+
+// Stream Functions
+let stream: fs.WriteStream;
+
+function streamWrite(str: string) {
+  stream.write(str + `\n`);
+}
+
+function startStream(filePath: string): fs.WriteStream {
+  const writableStream = fs.createWriteStream(filePath);
+
+  writableStream.on("error", (error) => {
+    console.log(
+      `An error occured while writing to the file. Error: ${error.message}`,
+    );
+  });
+  writableStream.on("finish", async () => {
+    const { fullPath, fileName } = getFilePathNameExt(filePath);
+    console.log(
+      `Converted toml config exists now under:${fullPath}/${fileName}.toml`,
+    );
+    const stream = fs.createReadStream(filePath);
+
+    const outJson = await toml.parse(await getStream(stream));
+    fs.writeFile(
+      `${fullPath}/${fileName}.json`,
+      JSON.stringify(outJson),
+      (error: any) => {
+        if (error) throw error;
+      },
+    );
+    console.log(
+      `Converted JSON config exists now under: ${fullPath}/${fileName}.json`,
+    );
+  });
+
+  return writableStream;
+}
+
+// TODO: Add support coonvertion for genesis customization / hrmpChannels and types
+async function convertInput(filePath: string) {
+  const { fullPath, fileName, extension } = getFilePathNameExt(filePath);
+
+  const convertedJson = await readInputFile(extension, filePath);
+
+  stream = startStream(`${fullPath}/${fileName}-zombie.toml`);
+
+  const { relaychain, parachains, simpleParachains } = convertedJson;
+
+  if (relaychain) {
```

Comment:
Hey @wirednkod, we can also simplify the conversion by work with `json` directly in js and the use something like [json2toml](https://www.npmjs.com/package/json2toml) to output `toml`. What do you think?

Reply:
🤔 Looks more decent than the solution below. Thank you @pepoviola - will make the convertion